client/src/components/EventTypeDialog.js

```diff
@@ -183,7 +184,7 @@ export default function EventTypeDialog() {
                             id="url"
                             label="URL"
                             type="text"
-                            InputProps={{ startAdornment: <InputAdornment position="start">calendapp.com/</InputAdornment> }}
+                            InputProps={{ startAdornment: <InputAdornment position="start">calendapp.com/{props.userUrl}/</InputAdornment> }}
```

Comment:
edit url input adornment, so it is now looks like this `calendapp.com/<username>/<eventType_url>/`